.github/workflows/gem-push.yml

```diff
@@ -0,0 +1,36 @@
+name: Ruby Gem
+
+on:
+  workflow_dispatch:
+  push:
+    tags:
+      - '*'
+
+jobs:
+  build:
+    name: Build + Publish
+    runs-on: ubuntu-latest
+    permissions:
+      packages: write
+      contents: read
+    strategy:
+      matrix:
+        type:
+          - jdbc
+          - mysql
+          - postgresql
+          - redshift
+          - sqlserver
+    steps:
+      - uses: actions/checkout@v2
+      - name: Set up Ruby 2.7
+        uses: ruby/setup-ruby@v1
+        with:
+          ruby-version: 2.7
+      - name: push gem
+        uses: t3t5u/push-gem-to-gpr-action@v1
```

Comment:
This will be fixed after the following PR merge.
https://github.com/trocco-io/push-gem-to-gpr-action/pull/3